### PR TITLE
Change windows docker image

### PR DESCRIPTION
**Pull request by Crola1702**
Co-authored by @claraberendsen

### CODE REVIEW

**Comment by clalancette** [comment]
Totally fine if you are using this for development purposes, but I just want to make sure we don't merge this change until after Iron is released.  Thanks!

**Comment by Crola1702** [comment]
Windows build with this image are passing https://github.com/osrf/packer-templates/issues/2#issuecomment-1535443790

**Review by nuclearsandwich — APPROVED** [review]

**Comment by ros-discourse** [comment]
This pull request has been mentioned on **ROS Discourse**. There might be relevant details there:

https://discourse.ros.org/t/ros-infrastructure-updates-from-may-2023https-discourse-ros-org-t-infrastructure-community-meeting-2023-05-17-31268-6/31759/1

**Pull request merged**
